.github/workflows/e2e_main.yaml

```
@@ -0,0 +1,71 @@
+# Copyright 2024 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: e2e
+
+env:
+  CI_COMMIT_REF_NAME: ${{ github.ref_name }}
+  GO_VERSION: "1.22.7"
+  GINKGO_VERSION: "2.20.0"
+
+on:
```

```suggestion
on:
  workflow_dispatch:
```
I think we may need to run this workflow manually.